docs/README.md

```diff
@@ -35,3 +35,28 @@ sphinx-autobuild . /tmp/inductiva-docs -W
 * `sphinx-autobuild . /tmp/inductiva-docs` will build the docs and watch for
 changes
 * `-W` will handle [warnings as errors](https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-W)
+
+## Deployment
+
+### Production
+
+Deployments to the production environment are **triggered automatically** whenever
+code is pushed to either `main` or `development` branches.
+
+These branches have different deployment targets:
+
+* `main` deploys to [latest](https://docs.inductiva.ai/en/latest/)
+* `development` deploys to [development](https://docs.inductiva.ai/en/development/)
+
+### Staging
+
+Deploying to the [staging environment](https://docs.staging.inductiva.ai/) is
+**performed manually** via GitHub actions.
+
+1. Go over to the [repository page](https://github.com/inductiva/inductiva/)
+2. Select the [Actions](https://github.com/inductiva/inductiva/actions) tab
+3. Select "Deploy Docs to Staging Server"
+4. Under "Run workflow ▼" select the branch you want to deploy and click "Run
```

Comment:
Does this really work now?
(was it already working before and I just did something stupid?)

Reply:
It should be working as-is.

E.g.: Selecting "main" and clicking "Run workflow" should deploy the main branch to staging.